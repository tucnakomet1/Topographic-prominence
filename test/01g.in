7604 7362 8373 8720 8708 7171 7375 8687 6768 7885 6673 6915 8563 6430 8283 7786 7745 7959 7345 7708 8656 7904 7092 7855 6596 8769 6036 6094 7489 8744 7086 6996 6713 6904 8667 8160 7253 7757 7705 7806 6187 6673 7175 6015 8817 8585 6128 6190 8226 8689