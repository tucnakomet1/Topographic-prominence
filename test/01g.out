242
2290
1516
111
1890
938
214
2060
763
2754
2708
1954
52
553
988
8817
2561